packages/jest-util/src/emptyTestResult.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {TestResult} from '@jest/test-result';
+
+export default function emptyTestResult(): TestResult {
```

Comment:
This seems to the minimal `TestResult` template that I could create

Reply:
Is there a legacy reason why `leaks` is mandatory, it looks like an optional field

Reply:
`leaks` should be optional, probably just an oversight. It's just a boolean though, right?